### ISSUE TITLE
Hello World - Python(docs/eventing/samples/helloworld/helloworld-python/_index.md) Trigger fails with BrokerDoesNotExist

### ISSUE THREAD

**Issue by boramalper** [knative/docs#3653]
## Expected Behavior
Following the instructions in [Hello World - Python](https://knative.dev/docs/eventing/samples/helloworld/helloworld-python/) sample tutorial of Knative Eventing should work.

## Actual Behavior
Step 2.3 of [Building and deploying the sample](https://knative.dev/docs/eventing/samples/helloworld/helloworld-python/#building-and-deploying-the-sample) fails. Trigger is not ready because `BrokerDoesNotExist`:

```bash
$ kubectl --namespace knative-samples get trigger helloworld-python
NAME                BROKER    SUBSCRIBER_URI   AGE   READY   REASON
helloworld-python   default                    60s   False   BrokerDoesNotExist
```

## Steps to Reproduce the Problem
- Follow the steps in [the tutorial](https://knative.dev/docs/eventing/samples/helloworld/helloworld-python/).

## Additional Info
**Additional context**
N/A

**Install information:**
- Platform (GKE, IKS, AKS, etc.): Self-Hosted Kubernetes, v1.20.7
- Knative Version:
  ```
  $ kn version
  Version:      v20210524-local-d0327bc
  Build Date:   2021-05-24 07:48:53
  Git Revision: d0327bc
  Supported APIs:
  * Serving
    - serving.knative.dev/v1 (knative-serving v0.23.0)
  * Eventing
    - sources.knative.dev/v1 (knative-eventing v0.23.0)
    - eventing.knative.dev/v1 (knative-eventing v0.23.0)
  ```

**Comment by boramalper** [comment]
The solution is to define a broker in `sample-app.yaml`.

```yaml
# Namespace for sample application with eventing enabled
apiVersion: v1
kind: Namespace
metadata:
  name: knative-samples
  labels:
       eventing.knative.dev/injection: enabled
---
# Helloworld-python app deploment
apiVersion: apps/v1
kind: Deployment
metadata:
  name: helloworld-python
  namespace: knative-samples
spec:
  replicas: 1
  selector:
    matchLabels: &labels
      app: helloworld-python
  template:
    metadata:
      labels: *labels
    spec:
      containers:
        - name: helloworld-python
          image: docker.io/{username}/helloworld-python
          imagePullPolicy: IfNotPresent
---
# Service that exposes helloworld-python app.
# This will be the subscriber for the Trigger
  kind: Service
  apiVersion: v1
  metadata:
    name: helloworld-python
    namespace: knative-samples
  spec:
    selector:
      app: helloworld-python
    ports:
    - protocol: TCP
      port: 80
      targetPort: 8080
---
# ======== FIX ========
# Create a Broker
apiVersion: eventing.knative.dev/v1
kind: Broker
metadata:
 name: default
 namespace: knative-samples
---
# Knative Eventing Trigger to trigger the helloworld-python service
apiVersion: eventing.knative.dev/v1
kind: Trigger
metadata:
  name: helloworld-python
  namespace: knative-samples
spec:
  broker: default
  filter:
    attributes:
      type: dev.knative.samples.helloworld
      source: dev.knative.samples/helloworldsource
  subscriber:
    ref:
      apiVersion: v1
      kind: Service
      name: helloworld-python
```

I am happy to send a PR if this is indeed considered a bug. [Hello World - Golang](https://knative.dev/docs/eventing/samples/helloworld/helloworld-go/) tutorial has a broker correctly defined.

**Comment by omerbensaadon** [comment]
@boramalper we would absolutely love to have that PR! Thank you for opening the issue and resolving it! 

PS: We would love to have you [join us on the Knative Slack](https://slack.knative.dev/)! Please reach out to me if you would like to get more involved in Knative :)

**Comment by omerbensaadon** [comment]
PPS: actually if you could DM me on that Slack I have a request for you!

**Comment by boramalper** [comment]
Fixed by #3682 I believe.
